src/panels/lovelace/badges/hui-entity-badge.ts

```diff
@@ -349,7 +349,7 @@ export class HuiEntityBadge extends LitElement implements LovelaceBadge {
         overflow: hidden;
       }
       .badge.no-info {
-        padding: 0;
+        padding: 0 0 0 6px;
       }
       .badge:not(.no-icon) img {
```

Comment:
```suggestion
        padding: 0;
      }
      .badge:not(.no-icon):not(.no-info) img {
```
It should be fixed this way.